From db0a65fa20a4182247e31c5fee02adaa46eaf075 Mon Sep 17 00:00:00 2001
From: kuangxiangbin <kuangxiangbin@contributor>


Date: Thu, 7 Aug 2025 11:42:14 +0800
Subject: [PATCH] BUGFIX open oma log

Issue:
------
open oma log, Record DDL logs, Help R&D solve the problem.
Solution:
---------
in deploy_package.sh set
rds_oma_log=ON
rds_oma_log_file=$mysql_log_dir/mysql_oma_log.txt
---
 BUILD/deploy_package.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/BUILD/deploy_package.sh b/BUILD/deploy_package.sh
index 941216fde4c..94ef91a8b50 100755
--- a/BUILD/deploy_package.sh
+++ b/BUILD/deploy_package.sh
@@ -171,6 +171,8 @@ loose-rpl_semi_sync_source_wait_point=AFTER_COMMIT
 replica_preserve_commit_order=ON
 slave_rows_search_algorithms='TABLE_SCAN,INDEX_SCAN'
 #agent managed
+rds_oma_log=ON
+rds_oma_log_file=$mysql_log_dir/mysql_oma_log.txt
 
 [mysqld_safe]
 pid-file=${mysql_pid_file}
-- 
2.23.0

